com.avaloq.tools.ddk.sample.helloworld/dsl-gen/com/avaloq/tools/ddk/sample/helloworld/validation/ExecutionEnvironmentStandaloneSetup.java

```diff
@@ -18,7 +18,7 @@ public class ExecutionEnvironmentStandaloneSetup implements ICheckValidatorStand
   private static final String GRAMMAR_NAME = "com.avaloq.tools.ddk.sample.helloworld.HelloWorld";
   private static final String CATALOG_FILE_PATH = "com/avaloq/tools/ddk/sample/helloworld/validation/ExecutionEnvironment.check";
 
-  @Override
+  /** {@inheritDoc} */

```

Comment:
the @override should stay, shouldn't it?

Reply:
Correct, two `dsl-gen` files were modified by mistake in this plugin. Fixed now.